combinations/hash.tsv

```diff
@@ -161,3 +161,5 @@ biopython=1.78,blast=2.9.0,pysam=0.17.0
 r-recetox-aplcms=0.9.3,r-base=4.1.0,r-arrow=4.0.1,r-dplyr=1.0.7	quay.io/bioconda/base-glibc-debian-bash:latest	0
 samblaster=0.1.26,samtools=1.14
 circularmapper=1.93.5,bwa=0.7.17
+fsnviz=0.3.0,crimson=1.0.0
```

Comment:
if you remove crimson, you should also get the latest crimson version, as crimson is a lower-pinned dependency of fsnviz